src/Lykke.Service.B2c2Adapter/Services/TradeHistoryService.cs

```diff
@@ -78,21 +117,18 @@ private async Task DoTimer(ITimerTrigger timer, TimerTriggeredHandlerArgs args,
             if (!StartWork())
                 return;
 
-            var offset = 0;
-            var page = 100;
-
             try
             {
                 using (var context = CreateContext())
                 {
-                    // Update trades from last time
-                    var data = await _b2C2RestClient.GetTradeHistoryAsync(offset, page, ct);
+                    var tradeRequest = new TradesHistoryRequest {Limit = 10};
```

Comment:
isn't 10 per page is too few?

Reply:
it was 10 in the previous implementation, so left it as is